tests/test_cmd_lossless.sh

```diff
@@ -68,6 +72,17 @@ pushd ${TMP_DIR}
   "${AVIFENC}" -y 400 -s 10 -l "${INPUT_GRAY_PNG}" -o "${ENCODED_FILE}"
   "${AVIFDEC}" "${ENCODED_FILE}" "${DECODED_FILE_LOSSLESS}"
   "${ARE_IMAGES_EQUAL}" "${INPUT_GRAY_PNG}" "${DECODED_FILE_LOSSLESS}" 0
+
+  # SVT-AV1 supports 4:2:0 lossless starting with version 3.0.0.
+  if "${AVIFENC}" --help | grep 'svt \['; then
+    # Depending on the version, either succeeds or prints an error message.
+    if "${AVIFENC}" -c svt --lossless --cicp x/x/16 -s 9 "${INPUT_Y4M}" "${ENCODED_FILE}" 2> "${OUT_MSG}"; then
```

Comment:
x/x/16 => 2/2/16

Note: I am surprised that the `x` does not result in an error. It was caused by the use of the `atoi()` function in `parseU32List()`.  https://en.cppreference.com/w/c/string/byte/atoi says "If no conversion can be performed, `​0​` is returned." I observed this in the debugger. "x/x/16" is parsed into {0, 0, 16}.

Another problem with `parseU32List()` is the use of the `strtok()` function. `strtok()` is not thread-safe. See https://en.cppreference.com/w/c/string/byte/strtok:

> Each call to `strtok` modifies a static variable: is not thread safe.

Current implementations of `strtok()` are likely to make it thread-safe by replacing the static variable with a thread-specific variable.